samples/accord/src/styles.scss

```diff
@@ -0,0 +1,5 @@
+// overwrite values
```

Comment:
it's weird to have two styles files. Why don't you move the css of accord to the scss file?

Reply:
Resolved! somehow I thought leaving it in the original css could be a sample of having both css and scss but I guess it doesn't really make sense.